Makefile

```diff
@@ -462,21 +463,22 @@ test-and-cover:
 		tail -n +2 profile_tmp.cov >> profile.cov; \
 	done
 
-android-lib: docker-golang-android genconfig
+android-lib-dev: docker-mobile
```

Comment:
Yes you are right. They converged to the point where they are very similar. The most important difference however is their dependencies. I made android-lib-dev at first to avoid running `genconfig`.